search/lib/api/stac.js

```diff
@@ -1,7 +1,66 @@
 const express = require('express');
+const cmr = require('../cmr');
+const cmrConverter = require('../convert');
+const { createRootCatalog, Catalog } = require('../stac').catalog;
+
+async function search (event, params) {
+  const cmrParams = cmr.convertParams(cmr.STAC_SEARCH_PARAMS_CONVERSION_MAP, params);
+  const granules = await cmr.findGranules(cmrParams);
+  return cmrConverter.cmrGranulesToFeatureCollection(event, granules);
+}
+
+async function getSearch (request, response) {
+  const event = request.apiGateway.event;
+  const params = cmr.convertParams(cmr.STAC_QUERY_PARAMS_CONVERSION_MAP, request.query);
+  const result = await search(event, params);
+  response.status(200).json(result);
+}
+
+async function postSearch (request, response) {
+  const event = request.apiGateway.event;
+  const result = await search(event, JSON.parse(request.body));
+  response.status(200).json(result);
+}
+
+async function getRootCatalog (request, response) {
```

Comment:
Just wondering if this function should be an asynchronous function as it doesn't await anything.

Reply:
I agree completely. I will remove this.